package.json

```diff
@@ -60,17 +59,18 @@
     "@typescript-eslint/parser": "5.55.0",
     "@vitejs/plugin-react": "3.1.0",
     "auto": "10.46.0",
-    "chromatic": "6.17.3",
```

Comment:
Why are we removing Chromatic? Is it included somewhere else?

Reply:
It runs using the GH action. The action installs the latest version of the CLI at all times.